src/pyhf/data/citation.bib

```diff
@@ -5,3 +5,16 @@ @software{pyhf
   doi = {10.5281/zenodo.1169739},
   url = {https://github.com/scikit-hep/pyhf},
 }
+
+@article{pyhf_joss,
```

Comment:
This should also appear in the README citation section with a clear request to cite both.